furusiya/model/maps/generators/map_generator.py

```diff
@@ -7,60 +7,46 @@
 
 
 def generate_monsters(area_map, num_monsters):
+    enemies = [
+        ('bushslime', config.data.enemies.bushslime, colors.desaturated_green),
+        ('steelhawk', config.data.enemies.steelhawk, colors.light_blue),
+        ('tigerslash', config.data.enemies.tigerslash, colors.orange)
+    ]
+    probabilities = [

```

Comment:
This should probably be drawn from `config.json`. And the values don't need to add up to 100.